subxt/src/utils/mod.rs

```diff
@@ -4,7 +4,10 @@
 
 //! Miscellaneous utility helpers.
 
+pub mod account_id;
```

Comment:
Not a big fan of these being grouped together in a module called `utils`, I think a more precise name would be better to indicate that these are primitive types from `sp` (substrate primitives) crates. Could just be `primitives`?

Reply:
Yeah, I was considering where to put them too. Really, they are sortof just "optional types that you can use in Config if you like", so they aren't explicitly a part of any API interface etc.

Perhaps they should be exposed in `config::substrate` for that reason?

Reply:
Yeah it's tricky, I think it's okay for misc stuff to go in `helpers`/`utils` namespace but in this case there is a clear category. Maybe `config::types`?

Reply:
If it helps, they are actually already exposed as `config::substrate::AccountId32` etc since they are used for substrate config, so they could also just be exposed from both places? And since they are used in codegen (in a way that I suppose isn't so directly related to the config?), maybe it's justifiable having them exposed in utils also?

But yeah if you prefer them living entirely in config::types or similar I'm happy with that too!